content/08-mongodb/14-mongodb-encryption.mdx

```diff
@@ -0,0 +1,83 @@
+
+MongoDB supports [AWS](https://docs.atlas.mongodb.com/security-aws-kms/#std-label-security-aws-kms), [Azure](https://docs.atlas.mongodb.com/security-azure-kms/#std-label-security-azure-kms), and [Google Cloud Platform](https://docs.atlas.mongodb.com/security-gcp-kms/#std-label-security-gcp-kms) key management services. This is a great feature for those who do not want to rely solely on MongoDB’s encryption keys and take ownership control of generated encryption keys.
+
+### Encryption in Use
+The last type of MongoDB encryption is a feature that provides encryption in use, [Client-Side Field Level Encryption (CSFLE)](https://www.mongodb.com/docs/manual/core/security-client-side-encryption/). CSFLE allows engineers to specify the fields of a [document(/intro/database-glossary#document) that should be kept encrypted. Sensitive data is encrypted/decrypted by the client and only communicated to and from the server in its encrypted form. 
+
+This feature ensures the security of encryption required fields on both the server and the network, accessible only by appropriate clients. At all times, sensitive data is encrypted on the database host and in network transit. Security responsibility shifts to the client. CSFLE prevents all of the vulnerabilities described throughout: database privilege abuse, network snooping, data theft, and access memory of a database host.
+
+## Rotating encryption keys
+There is always the risk of the symmetric key becoming compromised for encryption at rest and Client-Side Field Level encryption. Like you should do with your critical internet account passwords, you should also change your encryption keys regularly. This practice is rotating encryption keys.
+
+Encryption key rotation is crucial for database administrators to consider because while it is not surefire protection against a compromised key, it strongly avoids the risk. If a key gets compromised, then there is a likely chance that it has already become outdated for accessing sensitive data. It is another task to maintain, but it is a better practice to employ and never need than to be caught without doing it and wishing you had.
+
+MongoDB Atlas encryption at rest and CSFLE have built-in guides to rotate the required keys for each KMS cloud provider you might be using: [AWS KMS](https://docs.atlas.mongodb.com/tutorial/security-aws-kms-rotate-key/), [Azure Vault](https://docs.atlas.mongodb.com/tutorial/security-azure-kms-rotate-key/), and [Google KMS](https://docs.atlas.mongodb.com/tutorial/security-gcp-kms-rotate-key/). If you are self-hosting MongoDB, there are guides for conducting [manual key rotation](https://docs.mongodb.com/manual/tutorial/rotate-encryption-key/) and also MongoDB's [Ops Manager](https://docs.opsmanager.mongodb.com/current/tutorial/rotate-master-kmip-keys/index.html) and [Cloud Manager](https://docs.cloudmanager.mongodb.com/tutorial/rotate-master-kmip-keys/index.html) platforms. 
```

Comment:
These are all great links.  Pulling these out into a bulleted list is another option if you think it'd help them stand out better.